tests/test_job_management.py

```diff
@@ -500,15 +511,198 @@ def test_email_submitted(dispatcher_live_fixture, dispatcher_local_mail_server):
 
     # check the email in the email folders, and that the first one was produced
     assert os.path.exists(email_history_folder_path)
-    list_email_files = glob.glob(email_history_folder_path + '/email_1_submitted_*.email')
-    assert len(list_email_files) == 1
+    list_email_files = glob.glob(email_history_folder_path + '/email_submitted_*.email')
+    assert len(list_email_files) == 2
 
     assert os.path.exists(smtp_server_log)
     f_local_smtp = open(smtp_server_log)
     f_local_smtp_jdata = json.load(f_local_smtp)
 
     assert len(f_local_smtp_jdata) == 2
 
+    # let the interval time pass again, so that a new email si sent
+    time.sleep(16)
+    c = requests.get(server + "/run_analysis",
+                     dict_param
+                     )
+
+    assert c.status_code == 200
+    jdata = c.json()
+    assert jdata['exit_status']['job_status'] == 'submitted'
+    assert jdata['exit_status']['email_status'] == 'email sent'
+
+    # check the email in the email folders, and that the first one was produced
+    assert os.path.exists(email_history_folder_path)
+    list_email_files = glob.glob(email_history_folder_path + '/email_submitted_*.email')
+    assert len(list_email_files) == 3
+
+    assert os.path.exists(smtp_server_log)
+    f_local_smtp = open(smtp_server_log)
+    f_local_smtp_jdata = json.load(f_local_smtp)
+
+    assert len(f_local_smtp_jdata) == 3
+
+
+@pytest.mark.not_safe_parallel
+def test_email_submitted_multiple_requests(dispatcher_live_fixture, dispatcher_local_mail_server):
```

Comment:
there is some repetition on inspecting emails, maybe it can be taken out into some common functionality (similar to these ask/loop functions). It will be then easier to read the individual tests and also they can be reused in plugins.
A minor suggestion, maybe not now.